### PR TITLE
Cmd utils testing

### PR DESCRIPTION
**Pull request by djkresta**
7 cmd_utils tests are added and made to pass.

The remaining 4 our outlined and will be made to pass in the next commit.

### CODE REVIEW

File: src/licensemanager2/test/workload_managers/test_cmd_utils.py [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+@fixture
+def license_booking_request():
+    """
+    A license booking request object for testing make_booking_request()
+    """
+    lbr = LicenseBookingRequest()
+    lbr.job_id = 100
+    lbr.bookings = []
+    return lbr
```
Comment (jamesbeedy) [inline src/licensemanager2/test/workload_managers/test_cmd_utils.py:56]:
```suggestion
    return LicenseBookingRequest(job_id=100, bookings=[])
```

File: src/licensemanager2/test/workload_managers/test_cmd_utils.py [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+
+@fixture
+def license_booking_request():
+    """
+    A license booking request object for testing make_booking_request()
+    """
+    lbr = LicenseBookingRequest()
+    lbr.job_id = 100
+    lbr.bookings = []
```
Comment (jamesbeedy) [inline src/licensemanager2/test/workload_managers/test_cmd_utils.py:55]:
```suggestion
```

File: src/licensemanager2/test/workload_managers/test_cmd_utils.py [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+    return ["TESTPRODUCT", "TESTFEATURE", 5]
+
+
+@fixture
+def license_booking_request():
+    """
+    A license booking request object for testing make_booking_request()
+    """
+    lbr = LicenseBookingRequest()
+    lbr.job_id = 100
```
Comment (jamesbeedy) [inline src/licensemanager2/test/workload_managers/test_cmd_utils.py:54]:
```suggestion
```

File: src/licensemanager2/test/workload_managers/test_cmd_utils.py [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+    """
+    return ["TESTPRODUCT", "TESTFEATURE", 5]
+
+
+@fixture
+def license_booking_request():
+    """
+    A license booking request object for testing make_booking_request()
+    """
+    lbr = LicenseBookingRequest()
```
Comment (jamesbeedy) [inline src/licensemanager2/test/workload_managers/test_cmd_utils.py:53]:
```suggestion
```

**Comment by jamesbeedy** [comment]
test

File: src/licensemanager2/workload_managers/slurm/cmd_utils.py [hunk 5]
```diff
@@ -226,6 +238,10 @@ async def scontrol_show_lic():
     )
 
     stdout, _ = await asyncio.wait_for(proc.communicate(), CMD_TIMEOUT)
+    if proc.returncode != 0:
+        msg = "Scrontrol process return value was different from 0"
```
Comment (heitorPB) [inline src/licensemanager2/workload_managers/slurm/cmd_utils.py:242]:
```suggestion
        msg = f"Scontrol process returned {proc.returncode}"
```

What about also showing in the logs the return code *and* the stdout/stderr? Knowing why the command failed is very important for debugging.

**Pull request closed without merging**